hack/build/build-functest.sh

```diff
@@ -14,13 +14,29 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
-set -e
+set -euo pipefail
 
-script_dir="$(cd "$(dirname "$0")" && pwd -P)"
-source "${script_dir}"/../config.sh
+export PATH=$PATH:$HOME/gopath/bin
+JOB_TYPE="${JOB_TYPE:-}"
+
+if [ "${JOB_TYPE}" == "travis" ]; then
```

Comment:
travis was disabled completely for the kubevirt organization, so you can probably simplify this if else

Reply:
I was looking at HCO script while doing this. 

That is great. I've removed that part